ansible/roles/host-lets-encrypt-certs-certbot/tasks/main.yml

```diff
@@ -114,8 +134,8 @@
     - "{{ _certbot_dir }}/config"
     - "{{ _certbot_dir }}/work"
     - "{{ _certbot_dir }}/logs"
-    - "{{ _certbot_dir }}/renewal-hooks"
-    - "{{ _certbot_dir }}/renewal-hooks/deploy"
+    #- "{{ _certbot_dir }}/renewal-hooks"
```

Comment:
These should be removed.

Reply:
Shouldn't this be part of the "let's encrypt fix" effort? Since other configs use that role?